### ISSUE TITLE
BUG - Error when onboarding service only (no product yet in service tree) 

### ISSUE THREAD

**Issue by ladonnaq** [Azure/azure-sdk-tools#7315]
Background:  New service, service tree onboarding for product not complete (blocked by completion of Cloud ring onboarding). 

I met with a new service team this morning.  The engineer attempted to onboard but received an error message.  He proceeded to schedule Info Meeting, which worked.  No email was ever sent for him. 

I tired to onboard and I also got an error message.  However, this time autogenerated email was sent to me only (b/c) I completed the onboarding.  Also, the triage work item for the service was created. 

Opening issue so you can investigate. I am not sure what the error message means and why the email was sent and triage work item created for me but not when the engineer attempted.  

Service tree link - https://microsoftservicetree.com/services/3f340d4f-92b5-4489-9f01-c52f0cc6b9e5/overview

![image](https://github.com/Azure/azure-sdk-tools/assets/67656762/60c6ef73-53eb-4566-86e5-fd5e520a70c2)

**Comment by maririos** [comment]
@JonathanCrd did you have a chance to look into this bug?

**Comment by maririos** [comment]
I haven't been able to repro this bug, Things I tried:

- Repro it in PPE (PPE works fine)
- Ask another member outside of admin permissions to onboard the product from scratch on PPE
- Can't repro it in Prod because the Release Planner service has a product and this specific service doesn't
- It doesn't look like the error is from a flow so cloud flow history is not good. I did find the record for when La Donna onboarded with the service info but nothing jumped out there
- Admin portal and looking at failure logs, but this turned out to be more about the solution than the individual app

It looks to be a permission thing in our DB but there are no logs on it.
My recommendation for now is if a user hits this error again, to communicate with us. We do a monitor session so we can get the logs and then proceed. @ladonnaq this means don't get them unblocked. Send them to us and we will help them get unblock
